Fix compilation error when enabling Wireguard and XDP (#25734)
cilium-team/cilium: Documentation/security/network/encryption-wireguard.rst

  key in its corresponding ``CiliumNode`` CRD when a worker node's public key
  changes, given that the worker node will be unable to do so itself.

>   N/S Loadbalancer traffic won't be encrypted when an intermediate node redirects
>   a request to another node where a selected backend is running,
>   with the following configuration:
> 
>   - LoadBalancer & NodePort XDP Acceleration
>   - Direct Server Return (DSR)

Reviewer: Edits for clarity, use present tense, use clear, plain language
```suggestion
  N/S load balancer traffic isn't encrypted when an intermediate node redirects
  a request to a different node with the following backend configuration:

  - LoadBalancer & NodePort XDP Acceleration
  - Direct Server Return (DSR)
```

Author: Thank you for your comment! I have updated the doc accordingly.
